### DISCUSSION TITLE
State of loss functions in smp

### DISCUSSION THREAD

**Discussion by cwerner** [qubvel/segmentation_models.pytorch#521] [Q&A]
Hi all.

I was wondering what the current state of the loss functions in `smp` is? I read in some issues that DiceLoss for multi-class is broken. Or was this an old problem? I'm on 0.2 now and will update to 0.2.1 soon...

Since I also want to include a `boundary loss` into a combined loss term I was wondering if I should get my loss code from another library (i.e. [SegLoss](https://github.com/JunMa11/SegLoss))...?

Any recommendations?
